language/move-prover/bytecode/src/packed_types_analysis.rs

```diff
@@ -150,7 +150,10 @@ impl<'a> TransferFunctions for PackedTypesAnalysis<'a> {
 }
 
 impl<'a> DataflowAnalysis for PackedTypesAnalysis<'a> {}
-impl<'a> CompositionalAnalysis for PackedTypesAnalysis<'a> {}
+impl<'a> CompositionalAnalysis<PackedTypesState> for PackedTypesAnalysis<'a> {
+    fn to_summary(&self, state: PackedTypesState) -> PackedTypesState {state}
```

Comment:
Run `cargo xfmt` on this source. You will otherwise get an error that this source is not correctly formatted from CI (you did not see this because the CI test bailed out early because of trailing whitespace).

You may also want top run `cargo xclippy` to see any linting errors, which will also prevent you from passing CI tests.

In IDEs one can configure that code is automatically formatted on save. Not sure how to do this with vim.